csharp/ql/src/semmle/code/csharp/PrintAst.qll

```diff
@@ -260,7 +261,10 @@ class ControlFlowElementNode extends ElementNode {
           controlFlowElement.getParent*()
       )
     ) and
-    not isNotNeeded(element.getParent+())
+    not isNotNeeded(element.getParent+()) and
+    // LambdaExpr is both a Callable and a ControlFlowElement,
+    // print it with the more specific CallableNode
+    not element instanceof Callable
```

Comment:
This is unrelated to this PR, but it was consistently failing locally and in the CI job too, so I included the fix here.